src/api-service/__app__/onefuzzlib/agent_authorization.py

```diff
@@ -66,6 +67,7 @@ def verify_token(
         return not_ok(token, status_code=401, context="token verification")
 
     if not is_authorized(token):
+        logging.error("rejecting token - %s - %s", req.url, token)
```

Comment:
"Rejecting request". 
The token is valid (azure is doing the verification) but the request is not.
We should also log the body of the request too.